waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -522,6 +544,172 @@ protected String getUsage() {
         }
     }
 
+    /**
+     * Use {@code Producer} command to create a producer client that
+     * submits transactions to Waltz server.
+     *
+     * Producer process is closed once total sum of high watermarks for active
+     * partitions reaches num-callbacks.
+     */
+    private static final class CreateProducer extends ClientCliBaseClass {
+        private static final String NAME = "create-producer";
+        private static final String DESCRIPTION = "Creates one producer";
+
+        private CreateProducer(String[] args) {
+            super(args);
+        }
+
+        @Override
+        protected void configureOptions(Options options) {
+            Option txnPerClientOption = Option.builder("npt")
+                    .longOpt("num-produced-txn")
```

Comment:
It's better to keep txnPerClientOption option same as in the `class Validate`  i.e. `txn-per-client`